src/common/ui/style/index.js

```diff
@@ -2,15 +2,25 @@ import options from '../../options';
 import './style.css';
 
 let style;
+const CACHE_KEY = 'cacheCustomCSS';
+
+const setStyle = (css) => {
```

Comment:
What about `(css = '')` so you won't need `css = css || ''` below?

Reply:
> because getItem returns null, not undefined.

That's why I use `css = css || ''`.

I prefer `localStorage.getItem` because it's more explicit for reading a value.